core/src/main/java/nl/nn/adapterframework/stream/PathMessage.java

```diff
@@ -41,4 +41,7 @@ private PathMessage(Path path, MessageContext context, boolean removeOnClose) {
 	public static PathMessage asTemporaryMessage(Path path) {
 		return new PathMessage(path, new MessageContext(), true);
 	}
+	public static PathMessage asTemporaryMessage(Path path, String charset) {
```

Comment:
Ik weet niet of het handig is nu deze methode al aan te maken. We moeten dan goed oppassen dat we deze methode niet zomaar gaan gebruiken wanneer de charset variable bestaat.

Misschien dat je is na kan denken over een 'MessageWriter' en 'MessageOutputStream'. Die beide eerst naar een buffer schrijven en wanneer deze 'te vol' is alles flushen naar disk.
Het is denk ik niet handig lees en schrijf functionaliteit te combineren, een methode `#toMessage()` zou bijvoorbeeld een message terug kunnen geven.

Reply:
Oh ik geloof niet dat het mijn bedoeling was om die mee te nemen.
Die haal ik er snel weer uit.